packages/react/.storybook/manager-head.html

```diff
@@ -1,17 +1,29 @@
-<meta name="keywords" content="IBM, design, system, Carbon, design system, Bluemix, styleguide, style, guide, components, library, pattern, kit, component, cloud, React, React.js">
-<meta name="description" content="This is the React implementation of the Carbon Design System. Carbon is a series of individual styles and components, that when combined make beautiful, intuitive designs.">
+<meta
+  name="keywords"
+  content="IBM, design, system, Carbon, design system, Bluemix, styleguide, style, guide, components, library, pattern, kit, component, cloud, React, React.js"
+/>
+<meta
+  name="description"
+  content="This is the React implementation of the Carbon Design System. Carbon is a series of individual styles and components, that when combined make beautiful, intuitive designs."
+/>
 
 <!-- Open Graph -->
-<meta property="og:title" content="Carbon Components React">
-<meta property="og:site_name" content="Carbon Components React">
-<meta property="og:description" content="This is the React implementation of the Carbon Design System. Carbon is a series of individual styles and components, that when combined make beautiful, intuitive designs.">
-<meta property="og:image" content="https://media.github.ibm.com/user/525/files/59e3bfde-b990-11e7-87ef-072e89a87719">
-<meta property="og:url" content="http://react.carbondesignsystem.com">
+<meta property="og:title" content="Carbon Components React" />
```

Comment:
`<meta>` tags are self-closing, right? Or am I going crazy 😂

Reply:
Yeah my prettier auto formatted it 👍 will fix